extension/persistence/eclipselink/build.gradle.kts

```diff
@@ -28,6 +28,11 @@ dependencies {
   implementation(libs.eclipselink)
   implementation(platform(libs.dropwizard.bom))
   implementation("io.dropwizard:dropwizard-jackson")
+  val eclipseLinkDeps: String? = project.findProperty("eclipseLinkDeps") as String?
+  eclipseLinkDeps?.let {
+    val dependenciesList = it.split(",")
+    dependenciesList.forEach { dep -> implementation(dep.trim()) }
```

Comment:
Should we add a validation here to prevent any unexpected behavior? For example, if users accidentally or intentionally put a `}`, `)` in the string, it may lead to weird gradle behaviors.  Something like this 
```
fun isValidDep(dep: String): Boolean {
    val depRegex = "^[\w.]+:'[\w.-]+:[\w.-]+'$".toRegex()
    return dep.matches(depRegex)
}
```